.github/workflows/test.yml

```diff
@@ -35,24 +35,10 @@ jobs:
         run: |
           sudo apt-get -y install libpq-dev
 
-      - name: Set up Ruby 2.6.0
-        uses: actions/setup-ruby@v1
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.6
```

Comment:
We can get rid of this because [ruby/setup-ruby](https://github.com/ruby/setup-ruby) uses our `.ruby-version` instead 😄

Reply:
much easier!